app/views/devise/mailer/email_changed.html.erb

```diff
@@ -0,0 +1,7 @@
+<p>Hello <%= @email %>!</p>
+
+<% if @resource.try(:unconfirmed_email?) %>
```

Comment:
OPTIONAL

- [ ] It is a good practice to have zero logic in the views directory. Create methods in the helpers directory and call the methods in the views directory instead.
Read this [article](https://medium.com/@acushla4real/rails-view-helpers-best-practices-guide-d7d7b6db3db4) for a head-start. Another great piece to read is this [article](https://brandonhilkert.com/blog/refactoring-logic-from-a-rails-view/)